crates/portalnet/src/overlay/service/manager.rs

```diff
@@ -1129,11 +1129,13 @@ impl<
                                 // If the fallback FINDCONTENT task fails, that's fine for now.
                                 // In the future, we might want to cycle through all available
                                 // fallback peers on an error.
-                                let _ = Self::fallback_find_content(
+                                if let Err(err) = Self::fallback_find_content(
                                     content_key.clone(),
                                     utp_processing,
                                 )
-                                .await;
+                                .await {
+                                    debug!(%err, content_key = ?content_key, "Fallback FINDCONTENT task failed, after uTP transfer failed");
```

Comment:
```suggestion
                                    debug!(%err, ?content_key, "Fallback FINDCONTENT task failed, after uTP transfer failed");
```
nit: I think this would work. Same at line 1164